### DISCUSSION TITLE
Other "themes"

### DISCUSSION THREAD

**Discussion by pitosalas** [squidfunk/mkdocs-material#6637] [Q&A]
I like the functionality of Material a lot. But I would like some other choices for the look of my site/blog. Has anyone collected any sharable themes or templates or html or css?

**Comment by kamilkrzyskow** [comment]
Hello @pitosalas,
a quick google search away points to https://www.mkdocs.org/user-guide/choosing-your-theme/#third-party-themes 
which in turn points to https://github.com/mkdocs/catalog#-theming

You can also customize the look of your documentation with CSS:
https://squidfunk.github.io/mkdocs-material/customization/#additional-css

For example: https://www.privacyguides.org/en/

**Comment by pitosalas** [comment]
Yeah I saw those. But I assumed that the theme suggestions for MkDocs in
general would not work with Material, because it is already a theme. Am I
missing it?

**Reply by kamilkrzyskow** [reply]
Yes, they swap out Material entirely, as they're different themes, and using their CSS won't work as well. 
Now I understand you want a catalogue of already styled websites or styling snippets that you can use with your Material theme.
Sorry for the misunderstanding then, I don't know of any managed catalogue that would meet the criteria.
